src/Agent.Worker/Build/BuildJobExtension.cs

```diff
@@ -35,7 +35,9 @@ public override string GetRootedPath(IExecutionContext context, string path)
         {
             string rootedPath = null;
 
-            if (SourceProvider != null && Repository != null)
+            if (SourceProvider != null &&
+                Repository != null &&
+                StringUtil.ConvertToBoolean(Repository.Properties.Get<string>("__VSTS_READY")))
```

Comment:
If the `__VSTS_READY` property was added by this PR, should we name it with a newer nomenclature, like `__AZP_READY`?